libs/common/src/sql-db-entities/payment/payment.entity.ts

```diff
@@ -42,12 +43,12 @@ export class PaymentEntity extends AuditableBaseEntity {
   })
   feeRecords!: FeeRecordEntity[];
 
-  static create({ currency, amountReceived, dateReceived, paymentReference, feeRecords, requestSource }: CreatePaymentParams): PaymentEntity {
+  static create({ currency, amount, dateReceived, reference, feeRecords, requestSource }: CreatePaymentParams): PaymentEntity {
     const payment = new PaymentEntity();
     payment.currency = currency;
-    payment.amountReceived = amountReceived;
+    payment.amount = amount;
     payment.dateReceived = dateReceived;
```

Comment:
Am guessing we're leaving `dateRecieived` as `dateReceived` to signal that it's not an outstanding payment for whatever reason

Reply:
`dateReceived` feels like it is the correct name as it is the payment the date was received - not the date the payment was added to the table. Leaving it as `date` seems like it would be more ambiguous to me